### ISSUE TITLE
Move Widgets into the .entities packages instead of nesting in WidgetUtil

### ISSUE THREAD

**Issue by DV8FromTheWorld** [DV8FromTheWorld/JDA#2294]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [X] I have checked the branches or the maintainers' PRs for upcoming features.

### Feature Request

Based on conversations, there is an opportunity for a small reorganization of code. 
Context: https://github.com/DV8FromTheWorld/JDA/pull/2273#discussion_r996193457

TLDR: We want to move the `Widget` related classes out of WidgetUtil and instead have them in the `.entities` folder.
Specifically, promote the nested `Widget` class to its own file. The other Widget-subclasses should remain subclasses of Widget.
PRs welcome

**Comment by RealYusufIsmail** [comment]
Will hopefully make a pr for this

**Comment by MinnDevelopment** [comment]
Done in [5.0.0-beta.1](https://github.com/DV8FromTheWorld/JDA/releases/tag/v5.0.0-beta.1)
